whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -624,39 +635,35 @@ private Domain createDomain(final RpslObject rpslObject, final String requestUrl
         domain.getEntitySearchResults().addAll(createContactEntities(rpslObject, requestUrl));
         return domain;
     }
+    private static VCard createContactVCard(final RpslObject rpslObject) {
+        return commonVcard(rpslObject).build();
+
+    }
 
     private static VCard createVCard(final RpslObject rpslObject) {
+        return commonVcard(rpslObject).addEmail(rpslObject.getValuesForAttribute(E_MAIL)).build();
+    }
```

Comment:
Why we need this method separately for email?  also can you rename to addEmailVCard instead?

Reply:
In that case can you rename it to createVcardWithEmail Honestly I would prefer you have just one function createVcard and where you need to set email use entity.getVcard.addEmail.

Also can you rename commonVcard to createVcard? as it is not common to all it is based on objectType

Reply:
as we talked in the standup I reverted the change of e-mail. We will tackle this in a separated task